moon.md

```diff
@@ -0,0 +1 @@
+I a moon a planet? Please help me out, I think it is relevant.
```

Comment:
A moon is not a planet, but since we have visited the moon before, I think we should include it.